src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/collection-select-overlay/main.js

```diff
@@ -163,13 +142,13 @@ define(function() {
         },
 
         /**
-         * initialize column navigation
+         * Start column navigation which displays the collections
          */
         startOverlayColumnNavigation: function() {
-            this.sandbox.dom.append(this.$columnNavigationContainer, '<div class="container"/>');
+            this.sandbox.dom.append(constants.columnNavigationSelector, '<div class="container"/>');
```

Comment:
@nnatter do you really need to search globally for the column-navigation container. Please use something lik 'this.$find' or 'this.$el.find'